_pytest/vendored_packages/pluggy.py

```diff
@@ -540,7 +540,7 @@ def add_hookcall_monitoring(self, before, after):
         of HookImpl instances and the keyword arguments for the hook call.
 
         ``after(outcome, hook_name, hook_impls, kwargs)`` receives the
-        same arguments as ``before`` but also a :py:class:`_CallOutcome`` object
+        same arguments as ``before`` but also a :py:class:`_CallOutcome <_pytest.vendored_packages.pluggy._CallOutcome>` object
```

Comment:
the real problem is, that we change the absolute path, there - the whole thing is incorrect unless we stop vendoring